CHANGELOG.rst

```diff
@@ -2,10 +2,24 @@
 Change history
 ==============
 
+2.2.1
```

Comment:
```suggestion
2.3.0
```

Since this release does add a new feature (dynamic pagination), let's make it a minor release